tribits/core/package_arch/TribitsPackageMacros.cmake

```diff
@@ -728,6 +728,8 @@ macro(tribits_package_create_all_libs_interface_library)
       endif()
     endforeach()
     #print_var(packageLibsInAllLibsList)
```

Comment:
Dead code ? Should we remove it ?

Reply:
Yes

Reply:
Can you put in a PR to fix this?

Reply:
https://github.com/TriBITSPub/TriBITS/pull/461